vuepress/docs/next/docs/consume/entando-apis.md

```diff
@@ -3,139 +3,80 @@ sidebarDepth: 2
-
-3.  Wait for the application to start.
-
-        [INFO] Started ServerConnector@1355c8be{HTTP/1.1, (http/1.1)}{0.0.0.0:8085}
-        [INFO] Started @66257ms
-        [INFO] Started Jetty Server
-
-4. Navigate to the Swagger UI in a browser at `/entando-de-app/api/swagger-ui.html`
-
-        http://localhost:[your port]/entando-de-app/api/swagger-ui.html
-    
-
-## APIs Overview
+4. To enable the Swagger UI, add the "SPRING_PROFILES_ACTIVE" environment variable. If the variable is already present, add `swagger` to its comma-delimited value list:
+```
+        - name: SPRING_PROFILES_ACTIVE
+          value: default,swagger
+```
 
-The Entando core exposes REST APIs for every action that can be taken in
-the App Builder environment. For example, you can use
-these APIs to create pages, create page templates or to add widgets to
-pages. The APIs can be used to support automation, testing, or
-integrations with external systems.
+5. (Optional) Reset the deployment `spec.replicas` back to 1 if it was scaled to "0" in the previous steps. Save the deployment to update.
 
-### API structure
+### Disable Swagger UI
 
-All of the APIs share a common top level structure. Each response will
-contain a top level entry for `errors`, `metadata`, and `payload`.
+Repeat the steps above, but in step 4, remove `swagger` from the value list.
 
-The `errors` will always contain code and a message string indicating an
-error condition in the request. The `metadata` section is used for
-paging, sorting, filtering and data that is distinct from the body. The
-body of each response is included in the `payload` section of the
-response and varies according to each API.
+## Find Your Client Secret
+You'll need your client credentials to execute the Entando APIs. 
 
-### Models
+1. Login into your Keycloak Administration Console. To find the Kubernetes credentials, see the [Entando Identity Management System](./identity-management.md) page.
 
-All of the model classes returned by the Entando core are annotated so that the model definition is included in the Swagger documentation. At the bottom of the Swagger page all of the model classes returned by the API endpoints can be found.
+2. From the left nav, go to `Clients`
 
-## Tutorial
+3. Select the desired client (e.g. in a quickstart environment this is `quickstart`)
 
-1. Access your application Swagger UI as discussed above
+4. Click on the `Credentials` tab to retrieve the secret 
 
-2. Click on the `Authorize` button in the upper right corner
+## Access the APIs on Swagger
+1.  To see the APIs, go to: 
+``` sh
+http://[YOUR-HOST-NAME]/entando-de-app/api/swagger-ui.html
+```
 
-3. Enter the client id and client secret in the open window and click `Authorize`
+2. Click on the Authorize button in the upper right corner.
 
-4. If you are redirected to the Entando login page, log in with your credentials (default are `admin`/`adminadmin`)
+3. Enter the client id and secret and click Authorize.
```

Comment:
same wrt period

Secret

ID should be capitalized unless it's the property itself, then should be backticked